st/clitests/auth_spec.py

```diff
@@ -178,6 +178,22 @@ def account_tests():
     '''
     load_test_config()
 
+    test_msg = "Create account s3deletetest for testing Account Deletion with ldap credentials"
```

Comment:
add negative scenerio 1. try deleting account with bucket in it 2. delete account with IAM users in it 3. delete account with invalid ldap credentials

Reply:
done